platform/android/app/src/main/res/values-de/strings.xml

```diff
@@ -20,6 +20,9 @@
     <string name="onboarding_slide4_body_text">Wir möchten gerne sicherstellen, dass Ihre Daten im polyPod vor fremden Zugriff sicher sind. Dafür bitten wir Sie jetzt, die Gerätesperre auch für den polyPod einzuschalten.</string>
     <string name="settings_title">Einstellungen</string>
     <string name="settings_about_section">Info</string>
+    <string name="settings_sec_section">SecurityXXX</string>
```

Comment:
Guess this is a "TODO", right?

Reply:
needing a German here, yep - not sure how you mark undone translations...with TODO?

Reply:
Well, whatever you do, clarify if this will be done later in this same PR, or if it's going to merged into main, use something that looks reasonable... This is XML, anyway, so you can use comments here too.

Reply:
I would suggest "Sicherheit" for "settings_sec_section".

Reply:
updated them :) Christian provided me the translations after approving the texts